### PR TITLE
Fix blockstorage volume online retyping

### PR DESCRIPTION
**Pull request by morozovalekseywot**
Fix changing the volume_type of the vkcs_blockstorage_volume resource when it is connected to a compute instance

### CODE REVIEW

File: vkcs/blockstorage/resource_volume_test.go [hunk 1]
```diff
@@ -301,3 +325,55 @@ resource "vkcs_blockstorage_volume" "volume_1" {
...
+  block_device {
+    boot_index       = 0
+    source_type      = "volume"
+    destination_type = "volume"
+    uuid             = vkcs_blockstorage_volume.bootable.id
+  }
+
+   network_mode  = "none"
+}
+`
```
Comment (paaanic) [inline vkcs/blockstorage/resource_volume_test.go:379]: Let's just render it with a {{.VolumeType}} argument. I think that's a bit too much to clone the whole configuration for the single parameter.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ description: |-
 # VKCS Provider's changelog
 
 #### v0.8.3 (unreleased)
+- Fix changing the volume_type of the vkcs_blockstorage_volume resource when it is connected to a compute instance
```
Comment (paaanic) [inline CHANGELOG.md:11]:
```suggestion
- Fix unexpected state error when updating volume_type of an attached vkcs_blocstorage_volume
```

**Review by paaanic — CHANGES_REQUESTED** [review]

File: vkcs/blockstorage/resource_volume_test.go (hunk 1 above)
Comment (morozovalekseywot) [reply vkcs/blockstorage/resource_volume_test.go:353]: done

File: CHANGELOG.md (hunk 2 above)
Comment (morozovalekseywot) [reply CHANGELOG.md]: done

**Review by paaanic — APPROVED** [review]

**Pull request merged**